config/settings.yml

```diff
@@ -929,10 +930,31 @@ hqva_mobile:
       - "patient/Location.read"
       - "patient/Organization.read"
       - "patient/Patient.read"
+      - "patient/Condition.read"
+      - "patient/Observation.read"
+      - "patient/Practitioner.read"
+      - "patient/Medication.read"
+      - "system/Endpoint.read"
 
 # settings va.gov uses related to lighthouse api's
 lighthouse:
   api_key: fake_key
+  veterans_health:
+    url: https://sandbox-api.va.gov
+    fast_tracker:
+      client_assertion_type: "urn:ietf:params:oauth:client-assertion-type:jwt-bearer"
+      api_scope:
+        - "launch launch/patient"
+        - "patient/Patient.read"
+        - "patient/Condition.read"
+        - "patient/Observation.read"
+        - "patient/Practitioner.read"
+        - "patient/MedicationRequest.read"
+        - "system/Endpoint.read"
+      aud_claim_url: "https://deptva-eval.okta.com/oauth2/aus8nm1q0f7VQ0a482p7/v1/token"
+      client_id: "0oaaxkp0aeXEJkMFw2p7"
+      grant_type: "client_credentials"
+      api_key: /srv/vets-api/secret/lighthouse_fast_track_api.key
```

Comment:
Could we try a relative path here (i.e. `secret/lighthouse_fast_track_api.key`)? The /srv/vets-api directory is going away soon and we're trying to migrate away from using it.

Reply:
As a total newbie to the platform's deployment process, I'm wondering what that means for how we're retrieving keys during deployment. Are you able to point me to a piece of the code that is configured how you'd prefer, and also has the corresponding devops configuration set up properly/working?

Reply:
@ericboehs just another ping to draw some attention to this - let me know what you think!

Reply:
You will need to create a PR in the devops repo to add these settings to all environments. We store secrets in parameter store. You can create a support request with OPS to have them help you store the secrets in prarmeter store once your devops PR is ready. The files you will need to change are:

- ansible/deployment/config/vets-api/prod-settings.local.yml.j2
- ansible/deployment/config/vets-api/dev-settings.local.yml.j2
- ansible/deployment/config/vets-api/staging-settings.local.yml.j2
- ansible/deployment/config/vets-api/sandbox-settings.local.yml.j2

Reply:
I do have a staging configuration update merged into the devops repo as of last week: https://github.com/department-of-veterans-affairs/devops/pull/10352/files 

Does that look right to you?

Reply:
Right now, I'm still waiting for Lighthouse production access (nothing I can do from my end but wait - I need the client id they've generated from our credentials we shared to them via JWK). If we need to wait to merge this in until we have the production config in the devops repo, then I'll be sure to update you and this PR once that is done.

Reply:
This should be good for now 👍 .